test/ShapeCrawler.Tests.Unit/FontTests.cs

```diff
@@ -295,20 +295,20 @@ public void LatinName_Setter_sets_font_for_the_latin_characters(IShape shape)
     [Test]
     [MasterShape("001.pptx", "Freeform: Shape 7", 18)]
     [SlideShape("020.pptx", 1, 3, 18)]
-    [SlideShape("015.pptx", 2, 61, 18)]
+    [SlideShape("015.pptx", 2, 61, 18.67)]
     [SlideShape("009_table.pptx", 3, 2, 18)]
     [SlideShape("009_table.pptx", 4, 2, 44)]
     [SlideShape("009_table.pptx", 4, 3, 32)]
     [SlideShape("019.pptx", 1, 4103, 18)]
     [SlideShape("019.pptx", 1, 2, 12)]
-    [SlideShape("014.pptx", 2, 5, 21)]
+    [SlideShape("014.pptx", 2, 5, 21.77)]
     [SlideShape("012_title-placeholder.pptx", 1, "Title 1", 20)]
-    [SlideShape("010.pptx", 1, 2, 15)]
+    [SlideShape("010.pptx", 1, 2, 15.39)]
     [SlideShape("014.pptx", 4, 5, 12)]
     [SlideShape("014.pptx", 5, 4, 12)]
     [SlideShape("014.pptx", 6, 52, 27)]
     [SlideShape("autoshape-case016.pptx", 1, "Text Placeholder 1", 28)]
-    public void Size_Getter_returns_font_size(IShape shape, int expectedSize)
+    public void Size_Getter_returns_font_size(IShape shape, double expectedSize)
```

Comment:
What about converting `expectedSize` into `decimal` type?

Reply:
Sadly, this doesn't seem possible. C# does not allow `decimal` as an [attribute parameter type](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/attributes#2224-attribute-parameter-types). NUnit doesn't seem to want to convert it in this case, either. Happy to hear if you have a solution for this!